cmake/std/PullRequestLinuxDriver-Test.sh

```diff
@@ -150,13 +150,16 @@ echo -e ""
 : ${BUILD_NUMBER:?}
 : ${WORKSPACE:?}
 
-source /projects/sems/modulefiles/utils/sems-modules-init.sh
-
 declare -i ierror=0
 #Have to keep loading git
-module load sems-git/2.10.1
-
-
+regex=".*(_cuda_).*"
+if [[ ! ${JOB_BASE_NAME:?} =~ ${regex} ]]; then
```

Comment:
I checked this regex test in a script and it works... but I'm not sure we should make an assumption that if the job has `_cuda_` in its name that it's on Ride.  This might lead to bugs down the line if we create a CUDA job later on for a machine that isn't Ride... I'd rather see something like:

```
if job_name doesn't have CUDA:
    load non-cuda environment
else if machine is Ride
    load git module for Ride
else
    throw a meaningful error and notify DevOPs
fi
```

@prwolfe @jwillenbring your thougs?

Reply:
There might be a better way to word it, but, as things currently are, I was trying more so to warn someone if they are to create a CUDA job, they should put it on Ride... as it is currently the only one that is configured to run the rest of the CUDA build script that is to follow.
If we change that, we'll change/remove the note.
But it doesn't _need_ to be there, I don't have an issue with removing it.

Reply:
Agreed - Thanks @ZUUL42

Reply:
Committed the additional check.